tests/models/test_dag.py

```diff
@@ -1735,6 +1744,16 @@ def test_replace_outdated_access_control_actions(self):
             dag.access_control = outdated_permissions
         assert dag.access_control == updated_permissions
 
+    def test_validate_params_on_trigger_dag(self):
+        dag = models.DAG('dummy-dag', schedule_interval=None, params={'param1': Param(type="string")})
+
+        with pytest.raises(ValueError):
```

Comment:
```suggestion
        with pytest.raises(ValueError, match="Invalid input for param param1: None is not of type 'string'"):
```

Probably, just to make sure it actually shows useful error